tests/keras/backend/reference_operations.py

```diff
@@ -316,6 +316,15 @@ def mean(x, axis=None, keepdims=False):
         return np.mean(x, axis=axis, keepdims=keepdims)
 
 
+def var(x, axis=None, keepdims=False):
+    if isinstance(axis, list):
```

Comment:
np.var doesn't accept multiple axis as a tuple? Is it really needed to iterate with a for loop?

Reply:
Right, that was the problem! I was following the convention from function `std` (which is incorrect). The test for `std` and `var` (multiple axis) because the numpy result was incorrect.

Reply:
I agree that some pieces of code would need some changes in this file :)